### PR TITLE
Gavatar disabled by default and initials icon prettyfied

### PR DESCRIPTION
**Pull request by pboguslawski**
## Proposed change

Gavatar should be disabled by default for privacy (user agent activity and
e-mail disclosure) and security reasons (fetching data from external service
is less secure than using only internal resources).

This mod disables Gavatar by default (still may be enabled in
SysConfig if needed) and prettifies icon with user initials in
toolbar which is displayed instead of avatar; before

image

after:

image

## Type of change
- '1 - 🐞 bug 🐞'            - Bugfix (non-breaking change which fixes an issue)

## Additional information

- Internal information leaking is bug so this PR is bugfix in our opinion.
- Icon change is not required for fix to work and may be removed (= ignore changes in Header.tt and Core.Header.css) from this PR.
- Author-Change-Id: IB#1108605

## Checklist

- [x] The code change is tested and works locally.(❗)
- [x] There is no commented out code in this PR.(❕)
- [ ] You improved or added new unit tests.(❕)
- [x] Local ZnunyCodePolicy run passes successfully.(❕)
- [x] Local unit tests pass.(❕)
- [ ] GitHub workflow ZnunyCodePolicy passes.(❗)
- [ ] GitHub workflow unit tests pass.(❗)

### CODE REVIEW

**Comment by rkaldung** [comment]
Hi @pboguslawski,

Please follow the [contributing guidelines](https://github.com/znuny/Znuny/blob/rel-6_0-dev/CONTRIBUTING.md) and use the pull request template. And also, please point out if this is a bug fix or a feature.

**Comment by pboguslawski** [comment]
Done.

File: Kernel/Output/HTML/Templates/Standard/Header.tt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 Informatyka Boguslawski sp. z o.o. sp.k., http://www.ib.pl/
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/Header.tt]:
```suggestion
```

File: var/httpd/htdocs/skins/Agent/default/css/Core.Header.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /*
 Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+Copyright (C) 2021 Informatyka Boguslawski sp. z o.o. sp.k., http://www.ib.pl/
```
Comment (dennykorsukewitz) [inline var/httpd/htdocs/skins/Agent/default/css/Core.Header.css]:
```suggestion
```

**Review by dennykorsukewitz — CHANGES_REQUESTED** [review]
Hi @pboguslawski

please remove the copyrights.
I will apply the other changes as they are. 
With the new design, the CSS part may change again.

🚀

**Comment by dennykorsukewitz** [comment]
Hi @pboguslawski,

and please resolve conflicts 👍🏼 

Thanks.

**Comment by pboguslawski** [comment]
> please remove the copyrights

Done.

**Comment by dennykorsukewitz** [comment]
> Done.

Thank you 👍🏼 

Could you please check this UnitTest:
That probably needs to be adjusted here...
```
+-------------------------------------------------------------------+
  /opt/otrs/scripts/test/Selenium/Agent/AgentPreferences.t:
+-------------------------------------------------------------------+
..................................................................................................................................................................................
 not ok 179 - Avatars disabled message found
..
 not ok 182 - Gravatar message found
....................................................
/opt/otrs/scripts/test/Selenium/Agent/AgentPreferences.t ran 234 test(s) in 48.397s.
```

Regards :rocket:

**Comment by pboguslawski** [comment]
Unit test fixed in c419370b9b0d9a69b25979ed914de779339aa183.

**Comment by dennykorsukewitz** [comment]
Hi

This branch has conflicts that must be resolved.

Thank you! :rocket:

**Comment by dignin** [comment]
Thanks for your patience. This PR is not acceptable as-is. There are two changes in one. Please separate the changes for the default setting and resubmit if desired. The change to the icon will not be accepted. Pleases modify this in your system if needed.  We also recommend packaging it. 😉

**Pull request closed without merging**